design-system/materials/internals/definition.yaml

```diff
@@ -278,14 +288,6 @@ decisionGroupsByTheme:
           description: ''
           choice: color-surface
 
-    fontFamily:
-      label: Font family
-      prefix: font-family
-      decisions:
-        font-family:
-          description: ''
-          choice: font-family-body
```

Comment:
This decision group does not make sense to keep, as we have and use the choices (also for the theme overrides).